GATEWAY/src/providers/devices.service.ts

```diff
@@ -68,6 +71,6 @@ export class DevicesService {
    */
   public clearDisconnectedDevices = (): void => { // TODO: Change name?
     const currentTime = (new Date()).getTime();
-    this.devices = this.devices.filter(device => device.lastDiscovered - currentTime < 60000);
+    this.devices = this.devices.filter(device => currentTime - device.lastDiscovered < 60000);
```

Comment:
Are you sarcastic?
I honestly can't tell since this comment is under change requests.
Even if you are or not, it did not work before, so i fixed it 😛